hydra-tui/src/Hydra/TUI.hs

```diff
@@ -87,6 +87,7 @@ data State
       , dialogState :: DialogState
       , feedback :: [UserFeedback]
       , now :: UTCTime
+      , pending :: Bool
```

Comment:
suggestion: Use a dedicated type eg `data Wait = Waiting  | NotWaiting` to avoid "boolean blindness"